DOCS/man/mpv.rst

```diff
@@ -371,6 +371,79 @@ To use this feature, you need to fill the ``menu-data`` property with menu
 definition data, and add a keybinding to run the ``context-menu`` command,
 which can be done with a user script.
 
+Image Bindings
+--------------
+
+Several key bindings are changed when viewing images:
+
+LEFT, RIGHT, UP, DOWN, h, j, k, l
+    Pan the image when it is larger than the window.
+
+Ctrl+LEFT, Ctrl+RIGHT, Ctrl+UP, Ctrl+DOWN, Ctrl+h, Ctrl+j, Ctrl+k, Ctrl+l
+    Align the image to one of the boundaries of the window when it is larger
+    than the window.
+
+= and -
+    Change the zoom.
+
++ and _
+    Change the zoom slowly.
+
+0
+    Reset the zoom.
+
+u
+    Toggle between showing the image unscaled in its original dimensions and
+    fitting it to the window.
+
+o
+    Fill the window with the image.
+
+s
+    Toggle between showing images for 5 seconds in a slideshow and keeping them
+    open forever.
```

Comment:
Why is this needed? Just pause the playback.

Reply:
It doesn't just pause, it also enables the slideshow at runtime if you started with --image-display-duration=inf. I think this is a common feature in image viewers.

Reply:
Moved `cycle-values image-display-duration 5 inf` from s to space, though both are fine. The reasoning is:

- Space is easier to press than s
- `set image-display-duration inf` is the same as pausing for images, so it is simpler to use `image-display-duration` only
- Frees up s for users to bind it to something else